content/millennium/r4/base/workflow.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Workflow | R4 API
+layout: api
+---
+
+# Individuals
```

Comment:
Should probably be `# Workflow`

Reply:
https://github.com/cerner/fhir.cerner.com/commit/775cdd410a50bbc809a19a30052924b93f1fd20f